ext_tables.php

```diff
@@ -1,7 +1,4 @@
 <?php
 defined('TYPO3') or die();
 
-(static function (string $_EXTKEY) {
-    // Register hooks into \TYPO3\CMS\Core\DataHandling\DataHandler
-    $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['t3lib/class.t3lib_tcemain.php']['processDatamapClass'][] = \Causal\Oidc\Hooks\DataHandler::class;
-})('oidc');
+$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['t3lib/class.t3lib_tcemain.php']['processDatamapClass'][] = \Causal\Oidc\Hooks\DataHandler::class;
```

Comment:
the anonymous function is of no use right now but was at least in line with `ext_localconf.php`. I don't really care, still, not a real need to "clean-up" :)

Reply:
I didn't want to clean it away in the ext_localconf yet. Since all ext_localconf.php files are scoped in a namespace now by Core, this is not needed anymore.